src/platform.scad

```diff
@@ -0,0 +1,62 @@
+module platform (platformSize)
+{
+    platformSize = platformSize;
+    platformHeight = 0.15*platformSize;             // Height of the platform with respect to the platformSize.
```

Comment:
Also make platform height (or at least the factor between platformSize and platformHeight) a parameter of `platform` module so that when assembling the components later, we can specify the z-offset of the hall as a function of platform height so that we can basically 'put it on top' of the platform.